Jenkinsfile

```diff
@@ -292,8 +292,7 @@ podTemplate(name: 'fedora-atomic-' + env.ghprbActualCommit,
                         pipelineUtils.sendMessage(messageFields['properties'], messageFields['content'])
                     }
                     currentStage = "ci-pipeline-atomic-host-tests"
-                    stage(
-                            currentStage) {
+                    stage(currentStage) {
```

Comment:
I fail to see how this was a syntax error?

Reply:
This is the only one I see with a new line like this, and it is the only one not executing its duffy task.  It is pretty much a shot in the dark.  We had this problem before, and it was because there was no rsyncResults call in the atomic-host-tests stage.  The task isn't being run again, and I am not positive as to why.

Reply:
Right, added commit to really fix it